Source/SwiftLintFrameworkTests/ASTRuleTests.swift

```diff
@@ -148,6 +148,10 @@ class ASTRuleTests: XCTestCase {
             ruleDescription: FunctionBodyLengthRule.description,
             location: Location(file: nil, line: 1, character: 1),
             reason: "Function body should span 40 lines or less: currently spans 41 lines")])
+        let longerFunctionBodyWithComments = "func abc() {" +
+            Repeat(count: 40, repeatedValue: " // this is a comment\n").joinWithSeparator("") +
+            "}\n"
+        XCTAssertEqual(violations(longerFunctionBodyWithComments), [])
```

Comment:
Sorry, I had overlooked that the tests is not enough.
Maybe it should pass following tests:
```swift
        let longFunctionBodyWithComments = "func abc() {" +
            Repeat(count: 40, repeatedValue: "\n").joinWithSeparator("") +
            "// comment only line should be ignored.\n" +
            "}\n"
        XCTAssertEqual(violations(longFunctionBodyWithComments), [])

        let longerFunctionBodyWithComments = "func abc() {" +
            Repeat(count: 41, repeatedValue: "\n").joinWithSeparator("") +
            "// comment only line should be ignored.\n" +
            "}\n"
        XCTAssertEqual(violations(longerFunctionBodyWithComments), [StyleViolation(
            ruleDescription: FunctionBodyLengthRule.description,
            location: Location(file: nil, line: 1, character: 1),
            reason: "Function body should span 40 lines or less: currently spans 41 lines")])

        let longFunctionBodyWithMultilineComments = "func abc() {" +
            Repeat(count: 40, repeatedValue: "\n").joinWithSeparator("") +
            "/* multi line comment only line should be ignored.\n*/\n" +
            "}\n"
        XCTAssertEqual(violations(longFunctionBodyWithMultilineComments), [])
        
        let longerFunctionBodyWithMultilineComments = "func abc() {" +
            Repeat(count: 41, repeatedValue: "\n").joinWithSeparator("") +
            "/* multi line comment only line should be ignored.\n*/\n" +
            "}\n"
        XCTAssertEqual(violations(longerFunctionBodyWithMultilineComments), [StyleViolation(
            ruleDescription: FunctionBodyLengthRule.description,
            location: Location(file: nil, line: 1, character: 1),
            reason: "Function body should span 40 lines or less: currently spans 41 lines")])
```